src/lib/components/primitives/message/input-message.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
```

Comment:
New generic input message component. It replaces the old error message component.